### PR TITLE
build: update docker-compose files

### PR DESCRIPTION
**Pull request by Kif-Fando**

### CODE REVIEW

**Review by tamirGer — DISMISSED** [review]

File: docker-compose.local.yml [hunk 1]
```diff
@@ -3,21 +3,54 @@ version: '3'
...
+          'CMD-SHELL',
+          'pg_isready'
+        ]
+      timeout: 45s
     environment:
       POSTGRES_DB: bc
       POSTGRES_USER: bc
       POSTGRES_PASSWORD: bc
+      PGUSER: bc
+      PGPASSWORD: bc
```
Comment (derevnjuk) [inline docker-compose.local.yml:21]: You already set username and password for the superuser. In such case, you need to change the healthcheck


```
test:
  [
    'CMD',
    'pg_isready',
    '-U',
    '${POSTGRES_USER}',
    '-P',
    '${POSTGRES_PASSWORD}',
  ]
```

File: docker-compose.local.yml [hunk 2]
```diff
@@ -3,21 +3,54 @@ version: '3'
...
     ports:
       - 5432:5432
     volumes:
     - ./pg.sql:/docker-entrypoint-initdb.d/pg.sql
 
+  nodejs:
+    build:
+      context: '.'
+      dockerfile: Dockerfile
+    container_name: nodejs
```
Comment (derevnjuk) [inline docker-compose.local.yml:31]: I suggest avoiding using custom container name

```suggestion
```

File: docker-compose.yml [hunk 3]
```diff
@@ -8,18 +8,12 @@ services:
       POSTGRES_DB: bc
       POSTGRES_USER: bc
       POSTGRES_PASSWORD: bc
+      PGUSER: bc
+      PGPASSWORD: bc
```
Comment (derevnjuk) [inline docker-compose.yml:12]: See comment above

File: docker-compose.local.yml [hunk 4]
```diff
@@ -3,21 +3,54 @@ version: '3'
...
+    restart: always
+    ports:
+     - '3000:3000'
+    deploy:
+      restart_policy:
+        condition: on-failure
+      resources:
+        limits:
+          cpus: 3.5
```
Comment (derevnjuk) [inline docker-compose.local.yml:40]: It has no effect. It only works when user uses stack or swarm

File: docker-compose.yml [hunk 5]
```diff
@@ -8,18 +8,12 @@ services:
...
-      test:
-        [
-          'CMD',
-          'pg_isready',
-          '-U',
-          '${POSTGRES_USER}',
-          '-P',
-          '${POSTGRES_PASSWORD}',
-        ]
+      test: ['CMD', 'pg_isready']
```
Comment (derevnjuk) [inline docker-compose.yml:16]: Please revert these changes

File: docker-compose.yml [hunk 6]
```diff
@@ -71,18 +67,12 @@ services:
...
-      test:
-        [
-            'CMD',
-            'pg_isready',
-            '-U',
-            '${POSTGRES_USER}',
-            '-P',
-            '${POSTGRES_PASSWORD}',
-        ]
+      test: ['CMD', 'pg_isready']
```
Comment (derevnjuk) [inline docker-compose.yml:75]: ditto

File: docker-compose.yml [hunk 7]
```diff
@@ -71,18 +67,12 @@ services:
       POSTGRES_DB: keycloak
       POSTGRES_USER: keycloak
       POSTGRES_PASSWORD: password
+      PGUSER: keycloak
+      PGPASSWORD: password
```
Comment (derevnjuk) [inline docker-compose.yml:71]: See comments above

File: docker-compose.yml [hunk 8]
```diff
@@ -105,13 +95,14 @@ services:
...
       PROXY_ADDRESS_FORWARDING: 'true'
+      KEYCLOAK_FRONTEND_URL: 'https://auth.brokencrystals.neuralegion.com/auth/'
     healthcheck:
       test:
         [
           'CMD',
           'curl',
           '-f',
-          'http://localhost:8080/auth/realms/brokencrystals/health/check/database',
+          'https://auth.brokencrystals.neuralegion.com/auth/realms/master',
```
Comment (derevnjuk) [inline docker-compose.yml:105]: You have to use the internal address 

```suggestion
          'http://localhost:8080/auth/realms/master',
```

**Review by derevnjuk — CHANGES_REQUESTED** [review]
